src/theme/themes/pfr.scss

```diff
@@ -48,7 +48,7 @@
       // radio-button-font-color: var(--ion-color-primary),
       ion-item-background: var(--ion-color-gray-light),
       // task-progress-bar-color: var(--ion-color-primary),
-      // checkbox-background-color: white,,,
+      // checkbox-background-color: white,,,,
```

Comment:
nit(non-blocking):
not sure why this file keeps getting more commas, guessing something to do with code auto-formatters? In any case would suggest discarding the change from this PR and trying to fix root issue separately (maybe just discard all these commented properties?)

Reply:
Yes, I think this is prettier adding a trailing comma, see this open issue: https://github.com/prettier/prettier/issues/7752
I'm reluctant to remove these commented lines, as they represent the variables that are possible to be overridden by a theme (not a great system, but useful for now until it is improved). 
I'll try updating out prettier version, as it [looks like a possible fix was included in v3.0.2](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#do-not-add-trailing-comma-for-grouped-scss-comments-15217-by-auvred).